cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -128,6 +128,13 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 	if stgType == "device" {
 		return p.ProvisionBlockDevice(opts, pvCASConfig)
 	}
+	alertlog.Logger.Errorw("",
```

Comment:
Does w stand for warning?
How about
```go
log.Alert(....)
```
Can we avoid `""` in the beginning

Reply:
'W' stands for "with flags" in uber/zap logging library

Reply:
We can wrap the original implementation in log.Alert(...) format, but we will lose the severity flag. Or we can pass the severity as a parameter to the log.Alert() call. For eliminating the " " (empty text) we need can wrap this up in our wrapper function (log.Alert()). Will take this up in the next immediate PR. Meanwhile, the team can experiment with the log based alerts if this PR gets merged. And we may need to change some more things and add a few other flags or fields if we require. I can make all these changes in the next PR and finalise on this feature once the integration with Director is done.